plugins/copr.py

```diff
@@ -462,7 +463,7 @@ def _guess_chroot(self):
             dist = linux_distribution()
         # Get distribution architecture
         distarch = self.base.conf.substitutions['basearch']
-        if any([name in dist for name in ["Fedora", "Fedora Linux"]]):
+        if "Fedora" in dist:
```

Comment:
Thank you for the patch;  what do you have in `dist` in particular?  In my case it is `('Fedora Linux', '38', '')` triplet, so `Fedora` is not there.

Reply:
In my case it's the same except the first string is 'Asahi Fedora Linux Remix'

Reply:
To be honest this check should work for 'Fedora Linux' also, as it has the 'Fedora' substring

Reply:
Maybe we need dist[0] there, what do you think?

```
triplet1 = ('Fedora', '31', '')
triplet2 = ('Asahi Fedora Linux Remix', '37', '')
triplet3 = ('Fedora Linux', '38', '')
triplet4 = ('Debian', '10', '')

if "Fedora" in triplet1[0]:
  print("praiskup1")
if "Fedora" in triplet2[0]:
  print("praiskup2")
if "Fedora" in triplet3[0]:
  print("praiskup3")
if "Fedora" in triplet4[0]:
  print("praiskup4")

```

This prints:

```
praiskup1
praiskup2
praiskup3
```

Reply:
Perhaps that would be better, yes.

Reply:
Btw., while you are taking a look ... testing `dnf5` and `dnf5-plugins` would be worth it in your remix; there's a different detection mechanism and it might be broken as well.

Reply:
Tested on Asahi and pushed, hopefully I get time to try out dnf5 in the next month or so